src/Symfony/Component/Yaml/Parser.php

```diff
@@ -436,6 +436,9 @@ private function doParse($value, $flags)
                     $value = '';
 
                     foreach ($this->lines as $line) {
+                        if ('' !== $line && '#' === ltrim($line)[0]) {
```

Comment:
We need to use `'' !== ltrim($line)`. Otherwise the following access may fail if the current line consists of online whitespaces.

Reply:
done :+1: